hazelcast/src/main/java/com/hazelcast/map/impl/MapContainer.java

```
@@ -172,8 +173,10 @@ public void initEvictor() {
         } else {
             MemoryInfoAccessor memoryInfoAccessor = getMemoryInfoAccessor();
             EvictionChecker evictionChecker = new EvictionChecker(memoryInfoAccessor, mapServiceContext);
-            IPartitionService partitionService = mapServiceContext.getNodeEngine().getPartitionService();
-            evictor = new EvictorImpl(mapEvictionPolicy, evictionChecker, partitionService);
+            NodeEngine nodeEngine = mapServiceContext.getNodeEngine();
+            IPartitionService partitionService = nodeEngine.getPartitionService();
+            int batchSize = nodeEngine.getProperties().getInteger(MAP_EVICTION_BATCH_SIZE);
```

should we fail fast if `batchSize` is under 1?